src/hotspot/share/utilities/powerOfTwo.hpp

```diff
@@ -123,8 +123,9 @@ inline T next_power_of_2(T value)  {
 // Find log2 value greater than this input
 template <typename T, ENABLE_IF(std::is_integral<T>::value)>
 inline T ceil_log2(T value) {
```

Comment:
Pre-existing: It's a mistake that this function's return type is T.  It should be int, like all the other log2
variants in this file. Maybe it should be near those.

Reply:
I agree, see JBS text